Add tips on swap (#5825)
cyberbotics/webots: docs/guide/advanced-configuration.md

For more informations about assets, see [here](installation-procedure.md#asset-cache-download).

**Note**: This is already implemented in the Docker images provided by Cyberbotics.

#### Increase the Number of Parallel Simulations Supported by the Server

The server can become saturated because both its CPU and GPU RAM are full.
What happens normally is that the GPU and CPU will grow in parallel as new simulations are added.
In most cases the GPU RAM will be saturated first.
Once it happens, the CPU RAM will take the share of the GPU and therefore fill up faster.
Once the CPU RAM is full, the swap will take over.
Once the swap is full, the computer will crash if we try to open more simulations.

##### Enable the Zram

> [Zram](https://en.wikipedia.org/wiki/Zram) is a Linus kernel module that will compress the least used parts of the RAM.

Reviewer: ```suggestion
[Zram](https://en.wikipedia.org/wiki/Zram) is a Linux kernel module that will compress the least used parts of the RAM.
```

It has the same role as swap but is normally faster.

To enable it:
```
sudo apt-get install zram-config
sudo service zram-config start
```

To check the zram installation:
```
cat /proc/swaps
```

It should display something like:
```
Filename                                Type        Size    Used   Priority
/dev/sda3                               partition   9215996 0      -1
/dev/zram0                              partition   755740  8104    5
/dev/zram1                              partition   755740  8004    5
/dev/zram2                              partition   755740  8120    5
/dev/zram3                              partition   755740  8064    5
```

#### Increase the Size of the Swap

Increasing the size of the swap will allow the server to support more simulations in parallel.

> However, once the swap fills up, it is possible that there may be a performance drop.

Reviewer: ```suggestion
However, when the swap is filling-up, you may observe a performance drop.
```
